x/gov/bucket.go

```diff
@@ -101,6 +101,51 @@ func (b *ProposalBucket) Update(db weave.KVStore, id []byte, obj *Proposal) erro
 	return nil
 }
 
+const (
+	indexNameElectorate = "electorate"
+)
+
+// ResolutionBucket is the persistence bucket for resolutions.
+type ResolutionBucket struct {
+	orm.IDGenBucket
+}
+
+func NewResolutionBucket() *ResolutionBucket {
+	b := migration.NewBucket(packageName, "resolution", orm.NewSimpleObj(nil, &Resolution{})).
```

Comment:
Hmmm... could we use the new ModelBucket here?

Reply:
I was thinking about that actually, this is just a sample draft of my changes.